Source/WebInspectorUI/UserInterface/External/CSSDocumentation/CSSDocumentation.js

```diff
@@ -313,15 +313,18 @@ CSSDocumentation = {
     },
     "align-content": {
```

Comment:
the changes in this file look unrelated, can you undo them?

Reply:
They were generated using Source/WebInspectorUI/Scripts/update-inspector-css-documentation. Should I add just the hunk related to font-synthesis?

Reply:
I don't see anything related to font-synthesis in these changes (everything seems to be in overrides.json).

Possibly we should run this script in a separate commit, but this seems like the wrong place to do so. cc @dcrousso @patrickangle

Reply:
oh my bad, I had the impression it was updated. I'm removing it and just keeping the override.